src/components/Search/components/SearchInput/components/SearchInputBar/SearchInputBar.tsx

```diff
@@ -1,7 +1,37 @@
+import { API } from "@/apis/Api"
 import * as S from "./SearchInputBar.Styles"
+import { useState } from "react"
+import { Search } from "@mui/icons-material"
 
 const SearchInputBar = () => {
-  return <S.SearchInputBarLayout>SearchInputBar</S.SearchInputBarLayout>
+  const [input, setInput] = useState("")
```

Comment:
[P3] 이런 네이밍은 어떨까요? 🤔
```suggestion
  const [searchKeyword, setSearchKeyword] = useState("")
```

Reply:
좋습니당! 👏